pep-0695.rst

```diff
@@ -740,6 +767,19 @@ compiler. This mangled name would be based on the qualified name of the
 generic class, function or type alias it was associated with. This approach
 was rejected because qualified names are not necessarily unique, which means
 the mangled name would need to be based on some other randomized value.
+Furthermore, this approach is not compatible with techniques used for
+evaluating quoted (forward referenced) type annotations.
+
+
+Lambda Lifting
+--------------
```

Comment:
If you have an acknowledgments section, make sure Jelle is mentioned (your conversation with him seems to have put you on the current track).